athena-google-bigquery/src/main/java/com/amazonaws/athena/connectors/google/bigquery/BigQueryRecordHandler.java

```diff
@@ -263,7 +263,13 @@ private void getTableData(BlockSpiller spiller, ReadRecordsRequest recordsReques
                 // data available.  If no sessions are available for an anonymous (cached) table, consider
                 // writing results of a query to a named table rather than consuming cached results
                 // directly.
-                Preconditions.checkState(session.getStreamsCount() > 0);
```

Comment:
if we were throwing an exception before; are we returning zero now? 

how would that work for non count queries; meaning `select * from table` but returns 0 rows?